### PR TITLE
sign in & sign up by email/password

### PR DESCRIPTION
**Pull request by Keith-Go**
## Ticket
- [sign in & sign up by email/password](https://trello.com/c/jNlIed9x/18-email-passwork)

## Video

https://github.com/luke-nguyen-goldenowl/join_me/assets/152830147/53a04c97-0214-40f2-93db-15e67677c1d1

### CODE REVIEW

File: lib/src/features/authentication/view/signin_view.dart [hunk 1]
```diff
@@ -43,7 +45,14 @@ class SigninView extends StatelessWidget {
       crossAxisAlignment: CrossAxisAlignment.stretch,
       mainAxisAlignment: MainAxisAlignment.start,
       children: [
-        const SignTitle('Login'),
+        const SizedBox(height: 30),
+        // const SignTitle('Login'),
+        Image.asset(
+          "assets/images/images/logo2.png",
```
Comment (luke-nguyen-goldenowl) [inline lib/src/features/authentication/view/signin_view.dart]: Use asset.gen.dart

File: lib/src/features/authentication/view/signup_view.dart [hunk 2]
```diff
@@ -36,7 +44,20 @@ class SignupView extends StatelessWidget {
       crossAxisAlignment: CrossAxisAlignment.stretch,
       mainAxisSize: MainAxisSize.min,
       children: [
-        const SignTitle('Sigup'),
+        // const SignTitle('Signup'),
+        Image.asset(
+          "assets/images/images/logo2.png",
```
Comment (luke-nguyen-goldenowl) [inline lib/src/features/authentication/view/signup_view.dart]: use assets.gen.dart

**Comment by Keith-Go** [comment]
resolved

File: lib/src/features/authentication/view/signin_view.dart [hunk 3]
```diff
@@ -48,7 +47,7 @@ class SigninView extends StatelessWidget {
         const SizedBox(height: 30),
         // const SignTitle('Login'),
         Image.asset(
-          "assets/images/images/logo2.png",
+          Assets.images.images.logo2.path,
```
Comment (luke-nguyen-goldenowl) [inline lib/src/features/authentication/view/signin_view.dart]: Use Assets.images.images.logo2.image()
=> Same for all image or svg in assets folder

**Pull request merged**